src/edit_virtual_machine.zig

```diff
@@ -881,3 +888,93 @@ fn set_buffer(buffer: []u8, value: []const u8) void {
         index += 1;
     }
 }
+
+fn delete_confirmation_modal() !void {
+    // Render a modal when deleting a VM
+    var confirmation_window = try gui.floatingWindow(@src(), .{
+        .modal = true,
+        .open_flag = &deleting_vm,
+    }, .{
+        .color_style = .window,
+        .min_size_content = .{ .w = 300, .h = 100 },
+    });
+    defer confirmation_window.deinit();
+
+    var confirmation_vbox = try gui.box(@src(), .vertical, .{
+        .expand = .both,
+        .padding = .{ .x = 20, .y = 20, .w = 20, .h = 20 },
+    });
+    defer confirmation_vbox.deinit();
+
+    // Render some text to confirm deletion
+    try gui.label(@src(), "Are you sure you want to delete this virtual machine?", .{}, .{
+        .expand = .horizontal,
+    });
+
+    // Confirmation text input
+    {
+        var confiration_input_vbox = try gui.box(@src(), .vertical, .{
+            .expand = .both,
+            .padding = .{ .x = 0, .y = 5, .w = 0, .h = 5 },
+        });
+        defer confiration_input_vbox.deinit();
+
+        try gui.label(@src(), "Enter the full name of the VM to confirm deletion:", .{}, .{
+            .expand = .horizontal,
+            .font = .{
+                .size = 8,
+                .name = "VeraIt",
+                .ttf_bytes = fonts.VeraIt,
+            },
+        });
+
+        var confirmation_input = try gui.textEntry(@src(), .{
+            .text = &deletion_confirmation_text,
+            .scroll_vertical = false,
+            .scroll_horizontal_bar = .hide,
+        }, .{
+            .expand = .horizontal,
+        });
+        defer confirmation_input.deinit();
+    }
+
+    // A horizontal box to render buttons
+    {
+        var confirmation_hbox = try gui.box(@src(), .horizontal, .{
+            .expand = .horizontal,
+            .margin = .{ .y = 5 },
+        });
+        defer confirmation_hbox.deinit();
+
+        // Cancel button
+        if (try gui.button(@src(), "Cancel", .{ .expand = .horizontal, .color_style = .accent })) {
+            deleting_vm = false;
+            std.debug.print("Cancelled deletion of VM: {s}\n", .{vm.basic.name});
```

Comment:
Remove this debug print. By doing this you'll also be able to put the resulting code in one line.